src/uca/UserCollectableAttribute.js

```diff
@@ -86,7 +86,18 @@ const getAllProperties = (identifier, pathName) => {
       const typeDefDefinition = _.find(definitions, { identifier: typeDefinition.type });
       typeDefProps = resolveType(typeDefDefinition).properties;
     }
-    const basePropName = `${pathName ? `${pathName}.` : ''}${_.split(typeDefinition.identifier, ':')[2]}`;
+
+    let basePropName;
+    const baseIdentifierComponentes = _.split(typeDefinition.identifier, ':');
+    if (pathName) {
+      if (_.includes(pathName, _.lowerCase(baseIdentifierComponentes[1]))) {
```

Comment:
If I understood the logic right, you are getting all the properties of an UCA, but why use the LowerCase here? And this would clash with other namespaces no?

Reply:
In the claim path is all lower case.